src/api/controllers/messages.ts

```diff
@@ -102,5 +102,7 @@ export async function handleInternalMessageUpdate(data: DiscordPayload) {
     return;
   }
 
+  await structures.createMessage(payload);
```

Comment:
This does not save the member in the cache, just creates a member structure from the payload.